cmd/ci-operator/main.go

```diff
@@ -1380,6 +1385,64 @@ func printExecutionOrder(nodes []*api.StepNode) error {
 	return nil
 }
 
+type stepWithDependencies struct {
+	StepName     string
+	Dependencies []string
+}
+
+func dumpGraph(nodes []*api.StepNode) error {
+	artifactsEnv := os.Getenv(artifactsEnv)
```

Comment:
We have a `--artifacts` argument (which defaults to this), so we should use that.